cpu.c

```diff
@@ -1954,6 +1955,9 @@ static void __gb_step_cpu(struct gb_s *gb)
         (gb->gb_error)(gb, GB_INVALID_OPCODE, opcode);
     }
 
+#ifdef DEBUG
+    gb->gb_frame = 1;
```

Comment:
`gb_frame` should be initialized even if no `DEBUG` is defined.

Reply:
i trigger `gb->gb_frame` flag for `DEBUG` so that i can only run a step and return `cycles`
```c++
uint8_t gb_run_frame(struct gb_s *gb)
{
    uint8_t cycles = 0;
    gb->gb_frame = 0;

    while (!gb->gb_frame)
        cycles = __gb_step_cpu(gb);
    return cycles;
}
```